<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>媒体下载器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icon/icon/edit-media.png</normaloff>:/icon/icon/edit-media.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,10">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0,0,0,0,0,0,0,0,0">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="newButton">
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>新建任务</string>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icon/icon/download.png</normaloff>:/icon/icon/download.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="convertButton">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string> 转 码</string>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icon/icon/convert.png</normaloff>:/icon/icon/convert.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>暂停任务</string>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icon/icon/Stop (Filled).png</normaloff>:/icon/icon/Stop (Filled).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>删除任务</string>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icon/icon/delete.png</normaloff>:/icon/icon/delete.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="aboutButton">
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string> 关 于</string>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icon/icon/about.png</normaloff>:/icon/icon/about.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string> 退  出</string>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icon/icon/exit (1).png</normaloff>:/icon/icon/exit (1).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableWidget" name="downloadView">
        <property name="columnCount">
         <number>2</number>
        </property>
        <column/>
        <column/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>689</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
